tensorflow/python/ops/array_grad.py

```diff
@@ -853,7 +850,8 @@ def _ExtractImagePatchesGrad(op, grad):
       op.get_attr("padding"))
 
   # Create indices matrix for output tensor.
-  _, rows_out, cols_out, _ = [dim.value for dim in op.outputs[0].shape.dims]
+  output_bhwc = math_ops.cast(array_ops.shape(op.outputs[0]), ops.dtypes.int64)
```

Comment:
Same shape and dtype commentds apply here

Reply:
Thank you for your comment. I corrected the code as instructed.